src/exporter/DaemonMetricCollector.cc

```diff
@@ -192,8 +198,10 @@ void DaemonMetricCollector::update_sockets() {
       std::string daemon_name =
           daemon_socket_name.substr(0, daemon_socket_name.size() - 5);
       if (clients.find(daemon_name) == clients.end() &&
-          !(daemon_name.find("mgr") != std::string::npos)) {
+          !(daemon_name.find("mgr") != std::string::npos) && 
+          !(daemon_name.find("client") != std::string::npos)) {
```

Comment:
Don't remember if we are already using C++20 but it features a lot of Python-like methods, like [`.starts_with()`](https://en.cppreference.com/w/cpp/string/basic_string/starts_with).
```suggestion
          !(daemon_name.starts_with("mgr")) && 
          !(daemon_name.starts_with("client"))) {
```

Reply:
Well actually it may not start with mgr always so I'm just finding if mgr present. Anyways I'll be pushing 1 more PR for tls support so I can include some cleanups in it if necessary. Thanks!